src/yt_napari/_widget_reader.py

```diff
@@ -94,6 +97,7 @@ def remove_selection(self):
 
 class ReaderWidget(YTReader):
     _pydantic_model = _data_model.DataContainer
+    app = QApplication(sys.argv)
```

Comment:
These lines shouldn't be added -- napari takes care of all QT application and context handling and are why tests are stalling.
```suggestion
```